server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodServer.java

```diff
@@ -187,12 +189,18 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor e) {
       }
    };
 
+   private int getWorkerThreads() {
+      Integer override = Integer.getInteger(WORKER_THREADS_SYS_PROP);
+      return override == null ? configuration.workerThreads() : override;
```

Comment:
You can use
`Integer.getInteger(WORKER_THREADS_SYS_PROP, configuration.workerThreads());`